lib/wallet.js

```diff
@@ -54,6 +55,17 @@ function _balance (settings, cryptoCode) {
 function sendCoins (settings, toAddress, cryptoAtoms, cryptoCode) {
   return fetchWallet(settings, cryptoCode)
     .then(r => {
+      if (cryptoCode === 'BTC') {
+        return loadLatestConfig()
+          .then(config => {
+            const feeDiscount = config.wallets_BTC_feeDiscount
+            return r.wallet.sendCoins(r.account, toAddress, cryptoAtoms, cryptoCode, feeDiscount)
+              .then(res => {
+                mem.clear(module.exports.balance)
+                return res
+              })
+          })
+      }
```

Comment:
Can you check if this works? I pulled the .then outside
```suggestion
      if (cryptoCode === 'BTC') {
        return loadLatestConfig()
          .then(config => {
            const feeDiscount = config.wallets_BTC_feeDiscount
            return r.wallet.sendCoins(r.account, toAddress, cryptoAtoms, cryptoCode, feeDiscount)
          })
          .then(res => {
            mem.clear(module.exports.balance)
            return res
          })
      }
```